scripts/installation/bin/yb-ctl

```diff
@@ -636,6 +637,10 @@ class ClusterOptions:
         self.ysql_num_shards_per_tserver = args.ysql_num_shards_per_tserver
         self.timeout_yb_admin_sec = args.timeout_yb_admin_sec
         self.timeout_processes_running_sec = args.timeout_processes_running_sec
+        
```

Comment:
```suggestion

```

If you `git show --patch` your commit, you can see bright red where there are likely whitespace issues--assuming your git config is close to default.  I suggest also configuring your editor to show these whitespace issues if you haven't already.  There are two more bad whitespace areas, but I'll leave finding/fixing those as an exercise for you :slightly_smiling_face:.

Reply:
is there any auto-format for Python for VSCode?
for C++ previously I use `clang-format` but it reformated all the files, so I deactivate it.

Reply:
I have no particular recommendations.  You can see our lint rules in `.arclint`, but extraneous whitespace is, in general, not desired.